### PR TITLE
changes in network policy

### PR DESCRIPTION
**Pull request by ColinBe95**
## Description

See description in issue. Changing network policies following requirements from security team. Main changes are that zyte networkpolicy was moved to 0.0.0.0/0 and an additional mongodbnetworkpolicy to port 27017 on 0.0.0.0/0 was added.

Other changes are just removing duplicates and improving readability.

Discussed with @LukaszEV 

## Chart

Select the chart that you are modifying:
- [x] core
- [x] dotnet-core
- [ ] cron-job
- [ ] job
- [ ] app-reverse-proxy
- [ ] pact-broker

## Checklist
- [x] Description provided
- [x] Linked issue
- [x] Chart version bumped
- [ ] README.md updated with any new values or changes
- [ ] Updated template tests in `${CHART}/tests/*.py`

### CODE REVIEW

File: charts/core/templates/network-policy.yaml [hunk 1]
```diff
@@ -16,24 +16,27 @@ spec:
   ingress:
     - from: #any pod in current namespace
         - podSelector: {}
-  {{- if not .Values.global.allowAllInternalClusterTrafficNetworkPolicy }}
```
Comment (ColinBe95) [inline charts/core/templates/network-policy.yaml:19]: I only inverted this for readability. Can change it back if not desired

File: charts/dotnet-core/Chart.yaml [hunk 2]
```diff
@@ -4,6 +4,6 @@ name: charts-dotnet-core
 type: application
 dependencies:
 - name: charts-core
-  version: 2.0.12
+  version: 2.1.0
```
Comment (ColinBe95) [inline charts/dotnet-core/Chart.yaml:7]: should I update the version in `app-reverser-proxy`, `cron-job`, `job`, `pact-broker` as well?

File: charts/dotnet-core/templates/tests/deployment_test.py
Comment (ColinBe95) [inline charts/dotnet-core/templates/tests/deployment_test.py:1]: btw I did not change anything in this file, but apparently it was with CRLF line endings before. Now it is LF again.

**Review by AliakseiVenski — APPROVED** [review]

File: charts/dotnet-core/Chart.yaml (hunk 2 above)
Comment (ColinBe95) [reply charts/dotnet-core/Chart.yaml:7]: ah okay, so I first need to update the core charts then open another PR to update all depending charts? Will revert the change here then

Comment (ColinBe95) [reply charts/dotnet-core/Chart.yaml]: reverted

**Pull request merged**